src/pages/dashboard.vue

```
@@ -0,0 +1,15 @@
+<template>
+    <div>
+        Hello from dashboard
+    </div>
+</template>
+
+<script>
```

ESLint

[error] The 'lang' attribute of '<script>' is missing.

(vue/block-lang)